src/app/timeseries-menu.component.ts

```diff
@@ -62,7 +61,14 @@ export class TimeseriesMenuComponent {
   autoScale = true;
 
   private _model: Model;
-  private dateJson: any;
+
+  // returns the index of the lone level that is currently selected
+  lonerLevelCheck(): number {
```

Comment:
Can you please specify this as public/private? I'm assuming it should be public,